src/main/java/edu/depaul/cdm/se452/d2l_mock/course/Announcements/AnnouncementComponent.java

```diff
@@ -0,0 +1,30 @@
+package edu.depaul.cdm.se452.d2l_mock.course.Announcements;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
```

Comment:
Do we need this class for MS2 which only relates to persistence?

Reply:
This could not be merged as well.